smoke-test/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
-    "graphql": "16.5.0",
-    "make-fetch-happen": "10.2.1"
+    "graphql": "^16.5.0",
```

Comment:
This isn't a real published package so I kinda liked it being really explicit about what exactly is being tested (I'm pretty sure renovate works on it), but nbd.